scripts/clustering/clustering.py

```diff
@@ -7,29 +7,26 @@
 from rdkit.ML.Cluster import Butina
 from rdkit.Chem.AtomPairs import Pairs
 
-radius = 4
+radius = 2
 def compare_two_smiles(smile1, smile2, fp_type):
     smile1Ms = Chem.MolFromSmiles(smile1)
     smile2Ms = Chem.MolFromSmiles(smile2)
 
     if fp_type == "atom-pair":
         fps1 = Pairs.GetAtomPairFingerprintAsBitVect(smile1Ms)
         fps2 = Pairs.GetAtomPairFingerprintAsBitVect(smile2Ms)
-    elif fp_type == "fpm":
```

Comment:
Why is this option being removed?

Reply:
Good question! Minte emailed me a second time to ask about removing the finger print mols option as per his research it doesn't have any basis in science (lol). However, he wants to add another type I just didn't have the time to do the research for this pr. I'm adding #131 to address that

Reply:
Oh nice, that sounds good!